src/components/message/message.twig

```diff
@@ -0,0 +1,21 @@
+{# 
+  @TODO
+  1. Generate an example of a message with a decorative icon.
+  #}
+
+{% set tagName = tagName | default('div') %}
+
+<{{tagName}} class="c-message{% if class %} {{class}}{% endif %}">
+  {% block content %}
+  <p>
+    {{label | default('Hi, I\'m a message!')}}
+  </p>
+  {% endblock %}
+  {% if includeClose %}
+  <button aria-label="close" class="c-message__close u-pad-none" type="button">
```

Comment:
Why is `u-pad-none` a utility when all the other button styles are defined in `c-message__close`?

Reply:
@Paul-Hebert I feel like recently you shared some information on when to use `aria-label` versus visually hidden text? Am I imagining that?

Reply:
I think visually hidden text within the button would be preferable here.

The primary downside of `aria-label` is that it's [not reliably translated into other languages](https://heydonworks.com/article/aria-label-is-a-xenophobe/). (I think the sotry with Google Translate has gotten better but still isn't great)

So a screen reader user reading our site in Spanish would read the button text as "close" instead of "cerrar." If we used visually hidden text instead they'd hear "cerrar."